python-sdk/tests/benchmark/synthetic_parquet_generator.py

```diff
@@ -7,57 +7,50 @@
 fake = Faker()
 
 
-def generate_data(file_name, records):
+def generate_data(file_name, start, end):
     # Declare an empty dictionary
-    customer = {}
+    customer = []
     # Iterate the loop based on the input value and generate fake data
-    for n in range(0, records):
-        customer[str(n)] = {}
-        customer[str(n)]["id"] = str(n)
-        customer[str(n)]["name"] = fake.name()
-        customer[str(n)]["address"] = fake.address()
-        customer[str(n)]["email"] = str(fake.email())
-        customer[str(n)]["phone"] = str(fake.phone_number())
+    for n in range(start, end):
+        cust = {}
+        cust["id"] = str(n)
+        cust["name"] = fake.name()
+        cust["address"] = fake.address()
+        cust["email"] = str(fake.email())
+        cust["phone"] = str(fake.phone_number())
+        customer.append(cust)
 
     df = pd.DataFrame(data=customer)
-    df.to_parquet(file_name)
-
-    print(f"File {file_name} has been created.")
+    print(df)
 
+    df.to_parquet(file_name, index=None)
 
-# create ten_kb parquet file
-ten_kb = "/fake_dataset/parquet/ten_kb.parquet"
-ten_kb_rows = 13
-# generate_data(ten_kb,ten_kb_rows)
-
-# create hundred_kb parquet file
-hundred_kb = "/fake_dataset/parquet/hundred_kb.parquet"
-hundred_kb_rows = 140
-# generate_data(hundred_kb,hundred_kb_rows)
+    print(f"File {file_name} has been created.")
 
 
-# create hundred_mb json file
-hundred_mb = "/fake_dataset/parquet/hundred_mb.parquet"
-hundred_mb_rows = 15295
-generate_data(hundred_mb, hundred_mb_rows)
+five_gb = "/home/ankit_chaurasia/fake_dataset/parquet/five_gb.parquet"
```

Comment:
is it your local machine path?

Reply:
@pankajastro removed it in https://github.com/astronomer/astro-sdk/pull/978/commits/ae3ff922c2e4632287fdede5e5734e710a82510f